src/baldr/graphtile.cc

```diff
@@ -418,11 +418,14 @@ std::string GraphTile::FileSuffix(const GraphId& graphid, bool gzipped) {
   // it was something else
   stream << graphid.level() * static_cast<uint32_t>(std::pow(10, max_length)) + graphid.tileid()
          << ".gph" << (gzipped ? ".gz" : "");
-  return stream.str();
+  // now update path separator based on OS..
+  auto s = stream.str(); 
+  std::replace(s.begin(), s.end(), '/', filesystem::path::preferred_separator);
```

Comment:
i think instead of this you can go to the top of this file and change the separator there on line 31

Reply:
yes. will do.